### PR TITLE
fix: padding when keyboard is visible

### PR DESCRIPTION
**Pull request by yjkimjunior**
removes the RN KeyboardAvoidingView in favor of KeyboardAwareScrollView #268

### CODE REVIEW

**Comment by yjkimjunior** [comment]
IMG_0005
IMG_0006

File: src/screens/AccountNew.js [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 import React from 'react';
 import { StyleSheet, ScrollView, Text, View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+
```
Comment (Tbaut) [inline src/screens/AccountNew.js]: (nit) blank line misplaced I guess, should be under unstated

File: src/screens/AccountNew.js [hunk 2]
```diff
@@ -53,76 +55,78 @@ class AccountNewView extends React.Component {
...
-                this.props.navigation.navigate('AccountNetworkChooser')
-              }
-            >
-              <Text
+        <KeyboardAwareScrollView>
+          <Background />
+          <ScrollView
+            style={{ padding: 20 }}
+            keyboardDismissMode="on-drag"
+            keyboardShouldPersistTaps="handled"
```
Comment (yjkimjunior) [inline src/screens/AccountNew.js]:
```suggestion
            keyboardShouldPersistTaps="always"
```

File: src/screens/AccountRecover.js [hunk 3]
```diff
@@ -55,120 +57,122 @@ class AccountRecoverView extends React.Component {
...
-            }
+      <SafeAreaView style={styles.safeAreaView}>
+        <KeyboardAwareScrollView style={styles.bodyContainer}>
+          <Background />
+          <ScrollView
+            contentContainerStyle={{ justifyContent: 'flex-end' }}
+            style={{ flex: 1 }}
+            enableOnAndroid
+            scrollEnabled
+            keyboardShouldPersistTaps="handled"
```
Comment (yjkimjunior) [inline src/screens/AccountRecover.js]:
```suggestion
            keyboardShouldPersistTaps="always"
```

**Comment by hanwencheng** [comment]
tested on Iphone X (iOS simulator )
out

**Comment by hanwencheng** [comment]
tested on Iphone X (iOS simulator )
out

**Comment by hanwencheng** [comment]
Sorry, I have not elaborated more about the screenshots yesterday, but the behaviour on my Simulator is quite weird,  it seems the half of the text is still half-covered by the pop-up keyboard, it seems not to be an expected behaviour of KeyboardAwareScrollView according to the article [here](https://www.freecodecamp.org/news/how-to-make-your-react-native-app-respond-gracefully-when-the-keyboard-pops-up-7442c1535580/). Why is it like this @yjkimjunior , can we fix that?

**Pull request merged**